src/QMCWaveFunctions/tests/test_einset.cpp

```diff
@@ -169,7 +168,7 @@ const char *particles =
   REQUIRE(ddpsiV[1](0,1) == ComplexApprox(1.8089479397).compare_real_only());
   REQUIRE(ddpsiV[1](0,2) == ComplexApprox(0.5608575749).compare_real_only());
   REQUIRE(ddpsiV[1](1,0) == ComplexApprox(1.8089479397).compare_real_only());
-  REQUIRE(ddpsiV[1](1,1) == ComplexApprox(-0.07996207476).epsilon(eps).compare_real_only());
+  //REQUIRE(ddpsiV[1](1,1) == ComplexApprox(-0.07996207476).epsilon(eps).compare_real_only());
```

Comment:
Did this unit test problematic?

Reply:
Actually my system was giving a problem with catch.  ComplexApprox and epsilon were not working together and I could not figure out how to fix it.  If it's working for others it shouldn't stay commented out.

Reply:
When I build with the intel compiler, I get this:
/remote/lshulen/qmcpack/src/QMCWaveFunctions/tests/test_einset.cpp(171): error: class "Catch::ComplexApprox" has no member "epsilon"
    REQUIRE(ddpsiV[1](1,1) == ComplexApprox(-0.07996207476).epsilon(eps).compare_real_only());
    ^

/remote/lshulen/qmcpack/src/QMCWaveFunctions/tests/test_einset.cpp(171): error: class "Catch::ComplexApprox" has no member "epsilon"
    REQUIRE(ddpsiV[1](1,1) == ComplexApprox(-0.07996207476).epsilon(eps).compare_real_only());

and the build fails.  However gcc does not complain at all.  This was before I started working, on anything though.  I'll remove the comment in my next comment.